### PR TITLE
added repeatable script for test data closes #344

### LINKED ISSUE #344: CHORE > Create data loading script that will produce some demoable data (S)
DoD:
- [ ] SQL script exist to create:
  - [x] 10 member profiles with PDL relationships for all but 2
    - [x] 3 admins (me, geetika, and ron for now)
    - [x] 4 pdls (all the admins plus one more person from the team)
    - [x] the rest teammembers
  - [x] 3 teams
  - [x] 3 check-ins with action and agenda items for at least 2 members (a pdl and one of their team members)

### PR DESCRIPTION
**Pull request by hwilliams314**

### CODE REVIEW

**Comment by bhatiam12** [comment]
@hwilliams314 Build is failing - 

```CheckInControllerTest > testFindAllCheckInsByAdmin() FAILED
36
    org.opentest4j.AssertionFailedError at CheckInControllerTest.java:822
37

38
CheckInControllerTest > testGetFindByCompletedByAdmin() FAILED
39
    org.opentest4j.AssertionFailedError at CheckInControllerTest.java:755
40

41
RoleControllerTest > testFindRolesDoesNotExist() FAILED
42
    org.opentest4j.AssertionFailedError at RoleControllerTest.java:296
43

44
TeamControllerTest > testFindAllTeams() FAILED
45
    org.opentest4j.AssertionFailedError at TeamControllerTest.java:211
46

47
TeamMemberControllerTest > testFindAllTeamMembers() FAILED
48
    org.opentest4j.AssertionFailedError at TeamMemberControllerTest.java:281```

File: server/src/main/resources/db/R__Load_testing_data.sql [hunk 1]
```diff
@@ -0,0 +1,143 @@
+delete from action_items;
```
Comment (meyerinj) [inline server/src/main/resources/db/R__Load_testing_data.sql:1]: This script needs to be set up to not run in all environments. Should only run in local and dev. See https://github.com/oci-labs/check-ins/pull/297/files#diff-09185e226a8b5c7c467a55d4722011ed for examples of how to do this.

**Review by meyerinj — CHANGES_REQUESTED** [review]

File: server/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -41,3 +41,11 @@ credentials:
...
     MEMBER:
       - MEMBER
+---
+flyway:
+  enabled: enabled
+  datasources:
+    default:
+      locations:
+        - "classpath:db/common"
+        - "classpath:db/local"
```
Comment (mkimberlin) [inline server/src/main/resources/application-local.yml:51]: I think I would have called this second directory `db/dev`

File: server/src/main/resources/db/local/R__Load_testing_data.sql [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+VALUES ('517d324c-3d25-4ffe-887e-0de3ddc8fff9', 'PDL', '2c1b77e2-e2fc-46d1-92f2-beabbd28ee3d');
+
+INSERT INTO role(id, role, memberid)
+VALUES ('503fd79e-3e4d-4dc2-8045-498d329fd51f', 'PDL', '01b7d769-9fa2-43ff-95c7-f3b950a27bf9');
+
+INSERT INTO team(id, name, description)
+VALUES ('a8733740-cf4c-4c16-a8cf-4f928c409acc', 'Java Experts', 'Java Engineers of superior knowledge');
+
+INSERT INTO team(id, name, description)
+VALUES ('e8f052a8-40b5-4fb4-9bab-8b16ed36adc7', 'JavaScript GurusJavaScript Gurus', 'JavaScript Engineers of Outstanding Skill');
```
Comment (mkimberlin) [inline server/src/main/resources/db/local/R__Load_testing_data.sql:80]: DoubleJavaScriptGurus?

Comment (hwilliams314) [reply server/src/main/resources/db/local/R__Load_testing_data.sql:80]: yes, we are that good

File: server/src/main/resources/db/R__Load_testing_data.sql (hunk 1 above)
Comment (hwilliams314) [reply server/src/main/resources/db/R__Load_testing_data.sql]: done

File: server/src/main/resources/db/local/R__Load_testing_data.sql (hunk 3 above)
Comment (hwilliams314) [reply server/src/main/resources/db/local/R__Load_testing_data.sql:80]: done

**Review by meyerinj — APPROVED** [review]

**Review by bhatiam12 — APPROVED** [review]

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**